packages/clickhouse-server.service

```diff
@@ -17,6 +17,13 @@ User=clickhouse
 Group=clickhouse
 Restart=always
 RestartSec=30
+# The following ClickHouse directives should be used instead of forcing SIGKILL by systemd:
+# - shutdown_wait_unfinished_queries
+# - shutdown_wait_unfinished
+TimeoutStopSec=infinity
+# Disable forwarding signals by watchdog, since with default systemd's
+# kill-mode control-group, systemd will send signal to all process in cgroup.
+Environment=CLICKHOUSE_WATCHDOG_NO_FORWARD=1
```

Comment:
Actually this did not work with enabled watchdog, and this is the fix.
You will find descriptive commit message here about the reasoning - [`5b40c81` (#54744)](https://github.com/ClickHouse/ClickHouse/pull/54744/commits/5b40c81ac6bc7c545d942963550ed2347c267b58)